t/data/empty_test_distribution/needles/.gitkeep

```diff
@@ -0,0 +1,12 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use strict;
+use warnings;
+
+use testapi;
+use autotest;
+
+# intentionally not loading anything...
+
+1;
```

Comment:
I guess you copied that in by mistake. Likely .gitkeep should stay empty

Reply:
Guilty as charged.